src/components/Hero/ContentHero/ContentHero.stories.tsx

```diff
@@ -25,9 +26,17 @@ const meta = {
 export default meta
 
 export const ContentHero: StoryObj = {
+  parameters: {
+    // Set asPath in mock router so the Breadcrums component can render
+    // the "home" text with correct translation
+    nextjs: {
+      router: {
+        asPath: "/en",
+      },
+    },
+  },
   render: () => {
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    const { t } = useTranslation("page-learn")
+    const { t } = getI18n()
```

Comment:
Using this method is better here, so there are no errors/warnings of a hook being used outside of a component.